app/index.js

```diff
@@ -126,6 +126,10 @@ module.exports = yeoman.Base.extend({
     },
 
     gdtBase: function() {
+      // Save the existing composer.json file if any. See composerJson().
+      if (this.fs.exists('composer.json')) {
+        this.fs.copy('composer.json', 'composer.orig');
```

Comment:
Rather than perform this (virtual, check out https://github.com/gulpjs/vinyl-fs) file copy operation, why not load the values into memory and let the generator rewrite it as needed?